src/User32.Desktop/User32+PrintWindowFlags.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+
+    /// <content>
+    /// Contains the <see cref="PrintWindow"/> nested enum.
+    /// </content>
+    public partial class User32
+    {
+        public enum PrintWindowFlags : uint
+        {
+            // ReSharper disable InconsistentNaming
```

Comment:
We prefer the `SuppressMessage` notation to be able to provide an explicit reason. Most commonly:

```csharp
[SuppressMessage("ReSharper", "InconsistentNaming", Justification = "Original API names are used for consistency")]
```